packages/react-text/src/components/Text/Text.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import { makeStyles } from '@fluentui/react-make-styles';
+import * as React from 'react';
+import { Text } from './Text';
+import { TextProps } from './Text.types';
+
+const useStyles = makeStyles({
+  container: {
+    width: '100px',
+  },
+});
+
+export const TextStory = (props: TextProps) => {
+  const styles = useStyles();
+  return (
+    <div className={styles.container}>
+      <Text {...props}>This is an example of the Text component's usage.</Text>
+    </div>
+  );
+};
+
+TextStory.argTypes = {
+  wrap: {
```

Comment:
the controls should be generated from types, or did I not understand smth @Hotell  ?

Reply:
AFAIK this needs to be duplicated for compatibility reasons.